authorities.md

```diff
@@ -0,0 +1,95 @@
+# Authorities Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/integration/authorities**   
+
+Provide access to the configured authorities. It returns the list of existent authorities.
+
+Example: to be provided
+
+## Single Authority
+**/api/integration/authorities/<:authority-name>**
+
+Provide detailed information about a specific authority. The JSON response document is as follow
+```json
+{
+  "name": "SolrAuthorAuthority",
+  "hierarchical": false,
+  "scrollable": false
+}
+```
+
+Exposed links:
+* entries: the list of values managed by the authority
+
+## Linked entities
+### authority entries
+**/api/integration/authorities/<:authority-name>/entries **
+
+It returns the entries managed by the authority eventually filtered, see below 
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* metadata: the metadata that use the authority
+* query: the terms, keywords or prefix to search
+* parent: the key of the parent authority when searching in a hierarchical authority 
+* collection: the uuid of the collection where the item belong to
```

Comment:
We have added the parent parameter here because we want to replace the concept of XML vocabularies with the authorities. We are going to provide an authority implementation able to read the current XML vocabularies so that this functionality will be seen by the UI as a small extension of the authority feature instead than as a completely separate one